packages/TextEditor/src/text-editor.scss

```diff
@@ -0,0 +1,368 @@
+  font-family: Menlo, Consolas, Monaco, monospace;
+  font-size: var(--ids-text-editor-code-font-size);
+  line-height: var(--ids-text-editor-code-line-height);
+  margin: var(--ids-text-editor-code-margin);
+  overflow-x: auto;
+  padding: var(--ids-text-editor-code-padding);
+  position: relative;
+  tab-size: 2;
+}
+  
+.ids-text-editor__code::before {
+  background-color: var(--ids-text-editor-code-before-background);
+  border-right: 1px solid var(--ids-text-editor-code-before-border-color);
+  color: var(--ids-text-editor-code-before-color);
+  content: attr(data-gutter);
+  left: 0;
+  min-width: 25px;
+  padding: var(--ids-text-editor-code-before-padding);
+  position: absolute;
+  text-align: right;
+  top: 0;
+  white-space: pre-wrap;
+}
+
+.ids-text-editor__code::after {
+  color: rgb(0 0 0 / 50%);
+  content: attr(data-highlight-language);
+  font-size: var(--ids-text-editor-code-after-font-size);
+  padding: 3px;
+  position: absolute;
+  right: 3px;
+  text-transform: uppercase;
+  top: 0;
+}
+  
+.ids-text-editor__tokenComment {
+  color: var(--ids-text-editor-token-comment);
+}
+  
+.ids-text-editor__tokenPunctuation {
+  color: var(--ids-text-editor-token-punctuation);
+}
+  
+.ids-text-editor__tokenProperty {
+  color: var(--ids-text-editor-token-property);
+}
+  
+.ids-text-editor__tokenSelector {
+  color: var(--ids-text-editor-token-selector);
+}
+  
+.ids-text-editor__tokenOperator {
+  color: #9a6e3a;
+}
+  
+.ids-text-editor__tokenAttr {
+  color: #07a;
+}
+  
+.ids-text-editor__tokenVariable {
+  color: var(--ids-text-editor-token-variabl);
+}
+  
+.ids-text-editor__tokenFunction {
+  color: var(--ids-text-editor-token-function);
+}
+  
+.ids-text-editor__paragraph {
+  margin: var(--ids-text-editor-paragraph-margin);
+  position: relative;
+}
+  
+.ids-text-editor__paragraph:last-child {
+  margin-bottom: 0;
+}
+  
+.ids-text-editor__heading-h1 {
+  color: var(--ids-text-editor-heading-1-color);
+  font-size: var(--ids-text-editor-heading-1-font-size);
+  font-weight: var(--ids-text-editor-heading-1-font-weight);
+  margin: var(--ids-text-editor-heading-1-margin);
+  padding: 0;
+}
+  
+.ids-text-editor__heading-h2 {
+  color: var(--ids-text-editor-heading-2-color);
+  font-size: var(--ids-text-editor-heading-2-font-size);
+  font-weight: var(--ids-text-editor-heading-2-font-weight);
+  margin: var(--ids-text-editor-heading-2-margin);
+  padding: 0;
+}
+  
+.ids-text-editor__quote {
+  border-left: var(--ids-text-editor-quote-border-size) solid var(--ids-text-editor-quote-border-color);
+  color: var(--ids-text-editor-quote-color);
+  font-size: var(--ids-text-editor-font-size);
+  margin: var(--ids-text-editor-quote-margin);
+  padding: var(--ids-text-editor-quote-padding);
+}
+  
+.ids-text-editor__list-ol {
+  padding: 0;
+  margin: var(--ids-text-editor-quote-spacing);
+}
+  
+.ids-text-editor__list-ul {
+  padding: 0;
+  margin: var(--ids-text-editor-quote-spacing);
+}
+  
+.ids-text-editor__listitem {
```

Comment:
`list-item` or `listItem`?

Reply:
I'll go with list-item and I'll change the others to match.